box/converters.py

```diff
@@ -195,24 +207,22 @@ def _from_yaml(
     return data
 
 
-def _to_toml(obj, filename: Union[str, PathLike] = None, encoding: str = "utf-8", errors: str = "strict"):
+def _to_toml(obj, filename: Union[str, PathLike] = None):
```

Comment:
Can't change function signatures outside a major revision for Box, so will need to keep these as they were originally.

Reply:
I thought functions starting with underscore were supposed to be private.